templates/control-tower-customization.template.yml

```diff
@@ -42,29 +42,35 @@ Metadata:
 Parameters:
   NewRelicAccountNumber:
     Type: String
-    Description: New Relic account number.
+    NoEcho: true
+    Description: Your New Relic account ID. See https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/account-id/
```

Comment:
Please don't use "Your". It is self-explanatory that user needs to enter their account id and not someone elses ;)